lib/matplotlib/quiver.py

```diff
@@ -1009,7 +1009,8 @@ def _make_barbs(self, u, v, nflags, nbarbs, half_barb, empty_flag, length,
 
         *pivot* specifies the point on the barb around which the
         entire barb should be rotated.  Right now, valid options are
-        'head' and 'middle'.
+        'tip' and 'middle'. Can also be a number, which shifts the start
+        of the barb that many units from the origin.
```

Comment:
I'd have thought points, but the scatter size is defined in points. I don't remember the scaling, but if it's a square, then the radius is √49/2 = 3.5. If it's actual circular area, it's √(49/π) = 3.949. But neither of these seem to match the shift defined in the test of 1.7, so I'm still not sure.